src/composables/useIsInLobby.js

```diff
@@ -0,0 +1,59 @@
+/**
+ *
+ * @copyright Copyright (c) 2024, Dorra Jaouad <dorra.jaoued1@gmail.com>
+ *
+ * @author Dorra Jaouad <dorra.jaoued1@gmail.com>
+ * @author Daniel Calviño Sánchez <danxuliu@gmail.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { computed } from 'vue'
+
+import { useStore } from './useStore.js'
+import { PARTICIPANT, WEBINAR } from '../constants.js'
+
+/**
+ *
+ */
+export function useIsInLobby() {
+	const store = useStore()
+	const token = computed(() => store.getters.getToken())
+	const conversation = computed(() => store.getters.conversation(token.value))
+
+	const isModerator = computed(() => {
+		return conversation.value
+      && (conversation.value.participantType === PARTICIPANT.TYPE.OWNER
+        || conversation.value.participantType === PARTICIPANT.TYPE.MODERATOR
+        || conversation.value.participantType === PARTICIPANT.TYPE.GUEST_MODERATOR)
+	})
+
+	const isModeratorOrUser = computed(() => {
+		return conversation.value && !conversation.value.isDummyConversation && (isModerator.value
+      || conversation.value.participantType === PARTICIPANT.TYPE.USER
+      || conversation.value.participantType === PARTICIPANT.TYPE.USER_SELF_JOINED)
+	}).value
```

Comment:
```suggestion
	})
```
You break reactivity here by adding value (`computed.value` is boolean, so it doesn't return a reference to updated object).
It still can be used in Options API as `this.isModeratorOrUser`, so there will be no extra changes in the component

Reply:
cc @ShGKme for confirmation and double-check

Reply:
Yes, returning a value is returning a constant `true/false` value